### ISSUE TITLE
manim_studio\api.py, line 4, ImportError: cannot import name 'NoneType' from 'types' (C:\Python396\lib\types.py)

### ISSUE THREAD

**Issue by CitywideMars50** [MathItYT/manim-studio#21]
**Describe the bug**

```
C:\Users\hayde\Downloads\manim-test>manim-studio
Traceback (most recent call last):
  File "C:\Python396\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python396\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python396\Scripts\manim-studio.exe\__main__.py", line 4, in <module>
  File "C:\Python396\lib\site-packages\manim_studio\__init__.py", line 1, in <module>
    from .api import ManimStudioAPI, hold_on
  File "C:\Python396\lib\site-packages\manim_studio\api.py", line 4, in <module>
    from types import ModuleType, NoneType
ImportError: cannot import name 'NoneType' from 'types' (C:\Python396\lib\types.py)
```

**To Reproduce**
1. Install python 3.9.6
2. Open a cmd prompt as admin
3. run `pip install manim`
4. run `pip install manim-studio`
5. create project folder with template scene.py
6. run `manim-studio` in folder

could be because:
`decorator<6.0.0,>=5.0.7 in c:\python396\lib\site-packages (from manim<0.19.0,>=0.18.0->manim-studio) (5.1.1)`

**Expected behavior**
The app/gui should run

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: chrome
 - Version 21H2

**Smartphone (please complete the following information):** 
 - Device: why ??

**Additional context**

**Comment by MathItYT** [comment]
Hi! I'm just fixing that bug for the next release. It only happens in Python 3.9, I'm working to get it work for all compatible versions.

**Comment by MathItYT** [comment]
@d1ddle The new version is available! I tested it in Python 3.9 and it should be working now :)

**Comment by d1ddle** [comment]
Just got around to testing. Works like a charm. 😄 Closing Issue
